src/compiler/core.ts

```diff
@@ -994,11 +994,6 @@ namespace ts {
 
     /**
      * Gets the owned, enumerable property keys of a map-like.
-     *
-     * NOTE: This is intended for use with MapLike<T> objects. For Map<T> objects, use

```

Comment:
Comment outdated; `Map<T>` is now a real ES6 map, so we would use the `.keys()` method.